### ISSUE TITLE
a question in deepinsar.py

### ISSUE THREAD

**Issue by git97shunxin** [Lucklyric/DeepInSAR#1]
In line 134 of the program, should "clean_real" and "real" be replaced with "clean_imag" and "imag" respectively?

**Comment by Lucklyric** [comment]
Hi @git97shunxin , I really appreciate your head up. The bug has been fixed. 
Thank you!

**Comment by git97shunxin** [comment]
> Hi @git97shunxin , I really appreciate your head up. The bug has been fixed.
> Thank you!

Hi,is there still a problem in the loss function?The calculation method in the loss function does not seem to match the formula(9) in the paper.I would appreciate it if you could check this question.

**Comment by Lucklyric** [comment]
Hi @git97shunxin , 
```python
self.ae_r = self.batch_noisy_r[slice_index] - self.build_ae(
                self.features, is_train=True, reuse=reuse, name="dae_r", depth=self.config.fe_depth, ef_dim=self.config.fe_dim)
self.ae_i = self.batch_noisy_i[slice_index] - self.build_ae(
                self.features, is_train=True, reuse=reuse, name="dae_i", depth=self.config.fe_depth, ef_dim=self.config.fe_dim)
            
self.ae_loss_r = (1 / 2.0) * tf.reduce_mean(tf.reduce_mean(tf.squared_difference(self.ae_r, self.batch_clean_r[slice_index]), axis=[1, 2, 3]))
self.ae_loss_i = (1 / 2.0) * tf.reduce_mean(tf.reduce_mean(tf.squared_difference(self.ae_i, self.batch_clean_i[slice_index]), axis=[1, 2, 3]))
```
`self.batch_noisy_r` : noisy real channel, `self.batch_noisy_i`: noisy imag channel
`sef.build_ae(...)'s` outputs are estimated residuals for both channels. 
The loss of each channel is calculated as L2 of `((noisy - est_resdiual), clean)` for each channel.
